institution/fixtures/institutions.json

```diff
@@ -4,10 +4,21 @@
     "pk": 1,
     "fields": {
       "name": "Swansea University",
-      "base_domain": "swan.ac.uk",
+      "base_domain": "swansea.ac.uk",
```

Comment:
I will double check on this but I think that Swansea's Shibboleth IDP returns the users email address with the swan.ac.uk domain.

Reply:
I don't completely understand the Shibboleth integration and obviously can't test it directly, but I thought I remember seeing that the username provided by Shibboleth then got the base_domain appended to it so we were in control, although I don't see that code now on a cursory search.